internal/jbig2/bitmap/components_test.go

```diff
@@ -0,0 +1,1466 @@
+					// 10001001 00100000	- 0x89, 0x20
+					// 10001001 00100000	- 0x89, 0x20
+					// 10001001 00100000	- 0x89, 0x20
+					// 11110001 11100000	- 0xF1, 0xE0
+					data := []byte{0xF9, 0xE0, 0x89, 0x20, 0x89, 0x20, 0x89, 0x20, 0xF1, 0xE0}
+					assert.Equal(t, data, word.Data)
+					assert.Equal(t, box.Min.X, 2)
+					assert.Equal(t, box.Min.Y, 2)
+					assert.Equal(t, box.Max.X, 12+1)
+					assert.Equal(t, box.Max.Y, 6+1)
+				case 1:
+					// the second word should be like 'IT'
+					//
+					// 10011111	- 0x9F
+					// 10000100	- 0x84
+					// 10000100	- 0x84
+					// 10000100	- 0x84
+					// 10000100	- 0x84
+					data := []byte{0x9F, 0x84, 0x84, 0x84, 0x84}
+					assert.Equal(t, data, word.Data)
+					assert.Equal(t, box.Min.X, 18)
+					assert.Equal(t, box.Min.Y, 2)
+					assert.Equal(t, box.Max.X, 25+1)
+					assert.Equal(t, box.Max.Y, 6+1)
+				case 2:
+					// the third word should look like 'NOW'
+					//
+					// 10001001 11100100 01000000	- 0x89, 0xE4, 0x40
+					// 11001001 00100100 01000000	- 0xC9, 0x24, 0x40
+					// 10101001 00100101 01000000	- 0xA9, 0x25, 0x40
+					// 10011001 00100110 11000000	- 0x99, 0x26, 0xC0
+					// 10001001 11100100 01000000	- 0x89,	0xE4, 0x40
+					data := []byte{
+						0x89, 0xE4, 0x40,
+						0xC9, 0x24, 0x40,
+						0xA9, 0x25, 0x40,
+						0x99, 0x26, 0xC0,
+						0x89, 0xE4, 0x40,
+					}
+					assert.Equal(t, data, word.Data)
+					assert.Equal(t, box.Min.X, 30)
+					assert.Equal(t, box.Min.Y, 2)
+					assert.Equal(t, box.Max.X, 47+1)
+					assert.Equal(t, box.Max.Y, 6+1)
+				case 3:
+					//  the last component is the image of the 'skull'i
+					//
+					// 11111111 11110000 - 0xFF, 0xF0
+					// 11111111 11110000 - 0xFF, 0xF0
+					// 11000110 00110000 - 0xC6, 0x30
+					// 11000110 00110000 - 0xC6, 0x30
+					// 11000110 00110000 - 0xC6, 0x30
+					// 11111111 11110000 - 0xFF, 0xF0
+					// 00101010 10100000 - 0x2A, 0xA0
+					data := []byte{
+						0xFF, 0xF0,
+						0xFF, 0xF0,
+						0xC6, 0x30,
+						0xC6, 0x30,
+						0xC6, 0x30,
+						0xFF, 0xF0,
+						0x2A, 0xA0,
+					}
+
+					assert.Equal(t, 12, word.Width)
+					assert.Equal(t, 7, word.Height)
+					assert.Equal(t, data, word.Data, word.String())
+					assert.Equal(t, box.Min.X, 17)
+					assert.Equal(t, box.Min.Y, 13)
+					assert.Equal(t, box.Max.X, 28+1)
+					assert.Equal(t, box.Max.Y, 19+1)
+				}
+			}
+		}
+	})
+
+	t.Run("Zero", func(t *testing.T) {
+		// Create a zero like bitmap - it should not create any component.
+		bm := New(bm.Width, bm.Height)
+
+		bitmaps, boxes, err := bm.GetComponents(ComponentConn, 12, 8)
+		require.NoError(t, err)
+
+		if assert.NotNil(t, bitmaps) {
+			assert.Empty(t, bitmaps.Values)
+		}
+
+		if assert.NotNil(t, boxes) {
+			assert.Empty(t, *boxes)
+		}
+	})
+
+	t.Run("EmptyBitmap", func(t *testing.T) {
+		var b *Bitmap
+		_, _, err := b.GetComponents(ComponentConn, 12, 8)
+		assert.Error(t, err)
+	})
+
+	t.Run("NoEmptyFrame", func(t *testing.T) {
+		// t.Skip()
```

Comment:
Can we remove the commented code?
Typically we don't keep commented code unless for some reason, which should be indicated.